pkg/configuration/parameters.go

```diff
@@ -95,6 +98,16 @@ type JiraAPIResourceFlags struct {
 	ForceOnParent *bool
 }
 
+type JiraApiResourceParametersReadIssue struct {
+}
+
+type JiraApiResourceParametersEditCustomField struct {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type JiraApiResourceParametersEditCustomField should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4835527)